src/client/services/GoogleAnalyticsService.tsx

```diff
@@ -0,0 +1,64 @@
+import ReactGA from 'react-ga'
+
+const GA_TRACKING_ID = process.env.GA_TRACKING_ID || ''
+
+const GA_USER_EVENTS = {
```

Comment:
maybe use a typescript enum?

Reply:
Perhaps. On the other hand, now might be a good time to think through exactly how the GA events should be designed.

We have 4 groupings to think about - category, action, label, value. What can we map onto those?

Currently all GA events are categorised as `User`, with the only action available being `submit`. Could we perhaps incorporate more information from the submission into the event, perhaps the values of the outputs, and so on?

@lisatjide @lamkeewei - maybe something worth discussing here.